lightkurve/lightcurve.py

```diff
@@ -920,6 +922,17 @@ def bin(self, binsize=None, bins=None, method='mean'):
         - If the original lightcurve contains a quality attribute, then the
           bitwise OR of the quality flags will be returned per bin.
         """
+        # Early versions of astropy do not have calculate_bin_edges
+        try:
+            from astropy.stats import calculate_bin_edges
+            raise ImportError #for debugging purposes
+        except ImportError:
+            from astropy import __version__ as astropy_version
+            log.info("Version {} of astropy does not support calculate_bin_edges,"
+                  " falling back to numpy analog.  Upgrade astropy to version to"
+                  " >3.1 or >2.10 for consistent string input".format(astropy_version))
+            from numpy import histogram_bin_edges as calculate_bin_edges
```

Comment:
Mmmm I really think we should make a firm choice to use either the astropy or the numpy version in Lightkurve, and not fall back from one to the other, because that might create surprising behavior!

Reply:
Agreed!